output/components.eslint.txt

```diff
@@ -21,11 +21,14 @@ The react/require-extension rule is deprecated. Please use the import/extensions
   1:1  error  '@storybook/react' should be listed in the project's dependencies, not devDependencies  import/no-extraneous-dependencies
   4:2  error  Unexpected require()                                                                    global-require
 
+/home/travis/build/Talend/ui/packages/components/src/Table/Cell/TableCell.js
+  15:3  error  Visible, non-interactive elements should not have mouse or keyboard event listeners  jsx-a11y/no-static-element-interactions
```

Comment:
Please remove that error.

Reply:
Seen with @frassinier , we don't see why this error appears. The related code is:

<div
	className={classNames('tc-table-clickable-cell', className)}
	onClick={onClick}
	onDoubleClick={onDoubleClick}
	onKeyPress={onKeyPress}
	role="button"
	tabIndex="-1"
>
	{data}
</div>

According to this documentation [https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-static-element-interactions.md], it should be ok

Reply:
It's because you attach an onclick to a div. 
Just add and `eslint-ignore` comment.